### PR TITLE
REL-926456 add samples which utilise LoadfileNameFormat.WithDataSourc…

### PR DESCRIPTION
**Pull request by mateuszKulikRelativity**
added two files (*.cs and *.ps1) with samples that use LoadfileNameFormat.WithDataSourceNameAndDate

### CODE REVIEW

File: Relativity.Export.Samples.Powershell/SamplesCollection/export-folder-all-with-ds-name-in-loadfile.ps1 [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+. "$global:rootDir\Helpers\BaseExportService.ps1"
+
+# Your workspace ID: this is where we point to the workspace where we want to export from
+[int]$workspaceId = 1022188
+
+# Export settings parameters
+# Your view ID: view will provide us with available data to export, requires folder to be visible there.
+# Your folder ID: our targetted folder. If you want to export from the workspace root, the ID is different from the workspace ID.
+# ExportSourceType: ExportFolder or ExportFolderWithSubfolders
+[int]$viewId = 1003684
```
Comment (dawidstoga-rel) [inline Relativity.Export.Samples.Powershell/SamplesCollection/export-folder-all-with-ds-name-in-loadfile.ps1:12]: $viewId  -> $viewID

Comment (dawidstoga-rel) [reply Relativity.Export.Samples.Powershell/SamplesCollection/export-folder-all-with-ds-name-in-loadfile.ps1:12]: In some places there are 'Id' , in another 'ID' (uppercase)

Comment (mateuszKulikRelativity) [reply Relativity.Export.Samples.Powershell/SamplesCollection/export-folder-all-with-ds-name-in-loadfile.ps1:12]: this file is copied from other example, I simply added one line with "LoadfileNameFormat":"WithDataSourceNameAndDate",  but I get you

**Review by HueByte — APPROVED** [review]
LGTM

**Review by dawidstoga-rel — APPROVED** [review]

**Review by dawidstoga-rel — APPROVED** [review]

**Review by sylwesterpe — APPROVED** [review]

**Pull request merged**